WordPress/src/androidTest/java/org/wordpress/android/e2e/pages/MySitesPage.kt

```diff
@@ -280,4 +281,54 @@ class MySitesPage {
             }
         }
     }
+
+    private fun scrollToCard(elementID: Int): MySitesPage {
+        WPSupportUtils.waitForElementToBeDisplayed(elementID)
+        Espresso.onView(ViewMatchers.withId(elementID))
+            .perform(scrollTo())
+
+        return this
+    }
+
+    fun scrollToDomainsCard(): MySitesPage {
+        return scrollToCard(R.id.dashboard_card_domain_cta)
+    }
+
+    private fun tapCard(elementID: Int) {
+        WPSupportUtils.clickOn(elementID)
+    }
+
+    fun tapDomainsCard(): DomainsScreen {
+        tapCard(R.id.dashboard_card_domain_cta)
+        return DomainsScreen()
+    }
+
+    fun assertDomainsCard(): MySitesPage {
+        Espresso.onView(
+            Matchers.allOf(
+                ViewMatchers.withId(R.id.dashboard_card_domain_cta),
+                ViewMatchers.isDescendantOfA(ViewMatchers.withId(R.id.dashboard_cards)),
+                ViewMatchers.hasDescendant(ViewMatchers.withId(R.id.dashboard_domain_card_more)),
+                ViewMatchers.hasDescendant(ViewMatchers.withId(R.id.dashboard_card_domain_image)),
+
+                ViewMatchers.hasDescendant(
+                    Matchers.allOf(
+                        ViewMatchers.withText("Find a custom domain"),
```

Comment:
Is it fine to check a hardcoded string, "Find a custom domain"? It could be better to use resourceId for `"Find a custom domain"` and `"Stake your claim on your corner of the web with a site address that's easy to find, share, and follow."`. This way, it will also work when the string is modified, or when testing on a device that uses a different language.

Reply:
Thanks for this suggestion @irfano 🙇 I did this with 83ed21c3c783ad124949bb8f271713bf5fd926a0. The Instrumented Tests are ❌ because of [FTL service disruption](https://status.firebase.google.com/incidents/tzrfe4CYLYTLqsVyqWvy), I'll wait for it to be up again properly, and then will ask you for a re-review, after a rerun ✋